assets/src/hooks/useDetour.ts

```diff
@@ -213,14 +213,18 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
     /**
      * Removes the last waypoint in {@link waypoints} if {@link canUndo} is `true`.
      */
-    undo: state === DetourState.Finished ? undefined : undo,
+    undo,
```

Comment:
Dead code: `canUndo` is `false` in `DetourState.Finished`, which means that these functions are locked behind disabled buttons.